migrating/checklists/ossm-migrating-premigration-checklists-assembly.adoc

```diff
@@ -6,5 +6,121 @@ include::_attributes/common-attributes.adoc[]
+* You want to migrate from {SMProduct} 2 to {SMProduct} 3.
+* You are running {SMProduct} {SMv2Version}.
+* You have upgraded your `ServiceMeshControlPlane` resource to the latest version.
+* If you are using the {KialiProduct}, you are running the latest version.
+* You have installed the {SMProduct} Operator 3. To install the {SMProduct} 3 Operator, see: xref:../../install/ossm-installing-openshift-service-mesh.adoc[Installing OpenShift Service Mesh]
+
+[IMPORTANT]
+====
+You must complete the following checklists before you can begin migrating your deployment and workloads.
+====
+
+[id="disable-add-ons-and-reconfigure-replacements_{context}"]
+== Disable add-ons and reconfigure replacements
+
+* [ ] Disable Prometheus in your `ServiceMeshControlPlane` resource: `spec.addons.prometheus.enabled=false`
+** [ ] Configure the `ServiceMeshControlPlane` with OpenShift Monitoring as the replacement. These instructions also include installing a standalone `Kiali` resource. Both can be done at the same time. link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-integrating-with-user-workload-monitoring_observability[Integration user-workload monitoring]
+** [ ] If you are not using OpenShift monitoring, see: link:https://docs.redhat.com/en/documentation/red_hat_openshift_service_on_aws/4/html/service_mesh/service-mesh-2-x#integration-with-external-prometheus-installation[Integration with external Prometheus installation].
+
+* [ ] Disable tracing in your `ServiceMeshControlPlane` resource: `spec.tracing.type=None`
+** [ ] Configure the `ServiceMeshControlPlane` with OpenShift Distributed Tracing as the replacement: link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-configuring-distr-tracing-tempo_observability[Configuring {TempoName} and the Red{nbsp}Hat build of OpenTelemetry].
+
+* [ ] Disable Kiali in your `ServiceMeshControlPlane` resource: `spec.addons.kiali.enabled=false`
+** [ ] If you did not create a standalone `Kiali` resource as part of Prometheus or tracing, see: "Using {KialiProduct}".
+
+[id="migrate-to-explicitly-managed-routes_{context}"]
+== Migrate to explicitly managed routes
+
+Automatic route creation, also known as Istio OpenShift Routing (IOR), is a deprecated feature that is disabled by default for any `ServiceMeshControlPlane` resource created using {SMProduct} 2.5 and later. To move from {SMProduct} 2 to {SMProduct} 3, you need to migrate from IOR to explicitly-managed routes.
+
+If you already moved to explicitly-managed routes in {SMProduct} 2, then continue to gateway injection.
+
+* [ ] Migrate from Istio OpenShift Routing (IOR) to to explicitly-managed routes: link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-route-migration[Service Mesh route migration].
```

Comment:
Wondering why you use a link: here and not an xref, since its to service mesh, you should be able to xref, right?